### PR TITLE
fix(projects): #1622 show project logo if exist otherwise show projec…

### LINKED ISSUE #1622: Homepage project list to use project logo if exists and use place holder if not
Screenshot 2019-12-14 10 25 44

### PR DESCRIPTION
**Pull request by webkhushboo**
closes #1622 

 ## Notes

A summary of what was achieved in this PR

- [ ] Show project logo if exist on homepage public project list
- [ ] Show project logo if exist on homepage popular project list

Public project list:
image

Popular project list:
image

### CODE REVIEW

**Comment by webkhushboo** [comment]
@eddiejaoude 
Fixed the css for logo image on project dashboard page.

Before :
image


After:
With logo url:
image

Without logo url:
image

**Comment by eddiejaoude** [comment]
Profile images are now squashed 

Screenshot 2019-12-19 09 57 41


Screenshot 2019-12-19 09 57 55

File: web/src/app/main/components/homepage/homepage.component.html [hunk 1]
```diff
@@ -76,7 +74,10 @@
           <ng-container matColumnDef="icon">
             <th mat-header-cell *matHeaderCellDef></th>
             <td mat-cell *matCellDef="let element">
-              <mat-icon class="home__users-card__content__table__icon material-icons-outlined green-color">{{ checkTypeOfProject(element) }}</mat-icon>
+              <img *ngIf="element.logoUrl; else imagePlaceholder" src="{{element.logoUrl}}" />
```
Comment (eddiejaoude) [inline web/src/app/main/components/homepage/homepage.component.html]: It is better to have the src like `[src]="element.logoUrl"`

File: web/src/app/main/components/homepage/homepage.component.scss [hunk 2]
```diff
@@ -49,6 +49,10 @@
           height: 35px;
         }
 
+        img {
+          width: 24px;
+        }
+
```
Comment (eddiejaoude) [inline web/src/app/main/components/homepage/homepage.component.scss:55]: Effects the profile pictures. Comment and screenshots in PR conversation

File: web/src/app/main/components/homepage/homepage.component.html (hunk 1 above)
Comment (webkhushboo) [reply web/src/app/main/components/homepage/homepage.component.html]: okay

File: web/src/app/main/components/homepage/homepage.component.scss (hunk 2 above)
Comment (webkhushboo) [reply web/src/app/main/components/homepage/homepage.component.scss:55]: ok

Comment (eddiejaoude) [reply web/src/app/main/components/homepage/homepage.component.scss:55]: I made some updates to this already

File: web/src/app/main/components/homepage/homepage.component.html (hunk 1 above)
Comment (eddiejaoude) [reply web/src/app/main/components/homepage/homepage.component.html]: You did this in some places, I fixed the other places

**Comment by eddiejaoude** [comment]
I made some tweaks, @webkhushboo if you can review please when you get time.

The logo / icon here is not perfect, but I do not think it is a blocker for now

Screenshot 2019-12-19 12 25 34

**Review by webkhushboo — COMMENTED** [review]
Looks good 👍

**Comment by eddiejaoude** [comment]
Thank you 👍

**Review by eddiejaoude — APPROVED** [review]

**Pull request merged**